### DISCUSSION TITLE
Need EAR and ECCN information on WIX toolset usage

### DISCUSSION THREAD

**Discussion by lpentel1** [community/community#39672] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,

Please share ECCN code for WIX toolset license for the version WiX Toolset v3.11.2 and whether it subject to EAR, if any of you aware of it. Otherwise, kindly guide me on finding this information.

Regards,
Lakshmi

**Comment by david-engelmann** [comment, marked as answer]
[You can find info on the WIX toolset lincense here](https://wixtoolset.org/docs/about/license/). If you have further question, you will get the best responses by picking the relevant repo in [the Wix Toolset organization](https://github.com/wixtoolset/) and submitting your questions there!
